man/systemctl.xml

```diff
@@ -2025,7 +2030,11 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
             </listitem>
             <listitem>
               <para>when using <command>enable</command> or <command>disable</command> on units without
-              install information (i.e. don't have or have an empty [Install] section).</para>
+              install information (i.e. don't have or have an empty [Install] section),</para>
+            </listitem>
+            <listitem>
+              <para>when using <command>disable</command> combined with <option>--user</option> on units
```

Comment:
Spurious tags inserted. Please drop listitem, and para.

Reply:
I don't think this should be considered the same warning as the one for install information 🤔

Reply:
Ah, indeed. Sorry for the noise.